src/tyr/matrix_serializer.cc

```diff
@@ -43,6 +43,32 @@ json::ArrayPtr serialize_distance(const valhalla::Matrix& matrix,
   }
   return distance;
 }
+
+json::ArrayPtr serialize_shape(const valhalla::Matrix& matrix,
+                               const size_t start_td,
+                               const size_t td_count,
+                               const ShapeFormat shape_format) {
+  // TODO(nils): shapes aren't implemented yet in TDMatrix
```

Comment:
does this mean that we disable them further up in the code or that timedistance requests make it here and serialize an empty string?